### DISCUSSION TITLE
Digital Ocean install fails

### DISCUSSION THREAD

**Discussion by websirnik** [Stirling-Tools/Stirling-PDF#558] [General]
Hi, I'm trying to install Stirling-PDF using the "Deploy to DigitalOcean" button from ReadMe. I'm not changing any configurations, and the installation fails with the message "Git branch not found". I would appreciate any advice on how to deploy on Digital Ocean seamlessly. 
<img width="1002" alt="Screenshot 2023-12-25 at 21 43 43" src="https://github.com/Frooodle/Stirling-PDF/assets/282871/cf05c063-e97b-4563-83d2-fd8594abab51">

**Comment by Frooodle** [comment]
Ahh let me fix this tomorrow! It's a issue my side

**Comment by Frooodle** [comment]
Done a quick change, can you try now?
